kateryna/databases.py

```diff
@@ -0,0 +1,113 @@
+class PostgreSQLConnectionRefusedError(Exception):
+    def __init__(self, message):
+        self.message = message
+
+
+class Connection(object):
+    def __init__(self, db_name):
+        self.closed = False
```

Comment:
@pivancy The field is just used in a method that emulates connection closing. Or can i just write self.connection=None?